README.md

```diff
@@ -0,0 +1,14 @@
+## Running the app locally for development
+### Setup
+Run `docker-compose up`. This will create a Localstack and initialize it with everything under `./src/main/resources/localstack/`.
+Once started, admin service runs on `http://localhost:8089/`
```

Comment:
maybe worth mentioning need to build and run admin server first with 
{code}
mvn clean compile exec:java -Dvertex-configpath=conf/local-config.json
{code}

Reply:
Will do

Reply:
addressed